Resources/Prototypes/Entities/Structures/Windows/plastitanium.yml

```
@@ -77,6 +77,9 @@
   name: plastitanium window
   parent: PlastitaniumWindowSquareBase
   suffix: indestructible
+  components:
+    - type: ExaminableDamage
+      messages: null
```

Minor nitpick, but we like to deindent lists in yaml when we can (even though most editors autoindent them). Just saves a bit of linewidth.

```suggestion
  components:
  - type: ExaminableDamage
    messages: null
```

---

Fixed in both locations, will pay more attention to the style next time. Thank you!